notebooks/02-listas_y_for_loops.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Listas y for loops\n",
```

Comment:
<p>Está bueno mostrar que existen las listas y las tuplas y cómo se usa cada una!</p><p>Aunque la diferencia fundamental entre tuplas y listas es la inmutabilidad, no tanto qué tipo de variables podemos almacenar. Las listas son mutables:podemos agregarles y sacarles elementos, podemos modificar elementos ya presentes en ellas, etc. Las tuplas son inmutables: una vez creadas no podemos agregarles ni quitarles elementos, ni modificar elementos almacenados en ellas.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoLatinas/Intro-to-python/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>sip tambien podemos poner un ejercicio en como de pueden cambiar variables dentro de las listas pero no tuplas. Como si es mas complejo.. lo podeoms dejar para despues. Pero si hacer enfasis en usar corchetes :D  </p>